lib/std/math/gcd.zig

```diff
@@ -0,0 +1,41 @@
+/// Find the greatest common divisor (GCD) of two integers (uint), which are not all zero.
+/// For example, the GCD of 8 and 12 is 4, that is, gcd(8, 12) == 4.
+const std = @import("std");
+const expectEQ = std.testing.expectEqual;
+
+// Greatest common divisor (https://mathworld.wolfram.com/GreatestCommonDivisor.html)
+pub fn gcd(a: anytype, b: anytype) @TypeOf(a, b) {
```

Comment:
Looking at other files under `lib/std/math/`, the doc comment above the function here ought to go at the top of the file, and the doc comment at the top of the file (which, by the way, is erroneously documenting the `std` import) should go above the function here.

Thinking about it, one could argue that the comments at the top of the files in the math folder should be top level doc comments (`//!`).

Reply:
Yes, from my long experience we have always written these comments at the beginning of the source code. But as long as it is not defined otherwise, I move it to the function.